processing/pom.xml

```diff
@@ -99,6 +99,10 @@
             <groupId>commons-net</groupId>
             <artifactId>commons-net</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
```

Comment:
what uses this?

Reply:
I remember that I added it because Travis complain about it. But it seems no longer needed. Removed it now.

Reply:
Turns out I was using a util class in this library to create an array list from an iterator. Fixed it to use Guava instead.